cmd/exo/daemon.go

```diff
@@ -101,8 +101,14 @@ func loadRunState() error {
 }
 
 func newClient() *client.Root {
+	url := cfg.Client.URL
+	if url == "" {
+		url = runState.URL
+	}
+	url += "/_exo/"
```

Comment:
I made the change to include the `/` on the path rather than rely on a trailing slash on the URL itself, since this is more convential.

Reply:
That's ok by me, though I think I tried that at some point and backed it out for some reason. May have just been b/c it had knock-on effects and I was already in the middle of another change.

Reply:
I tried things out after the fact, and everything *seemed* to work, but it does make me a bit nervous to change.

Reply:
as long as the CLI and the GUI both work, should be fine